server/src/test/java/org/apache/iotdb/db/integration/IoTDBFlushQueryMergeIT.java

```diff
@@ -154,4 +154,24 @@ public void testFlushGivenGroup() throws ClassNotFoundException {
       fail(e.getMessage());
     }
   }
+
+  // bug fix test, https://issues.apache.org/jira/browse/IOTDB-875
+  @Test
+  public void testFlushGivenGroupNoData() throws ClassNotFoundException {
+    Class.forName(Config.JDBC_DRIVER_NAME);
+    try (Connection connection = DriverManager
+        .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+        Statement statement = connection.createStatement()) {
+      statement.execute("SET STORAGE GROUP TO root.nodatagroup1");
+      statement.execute("SET STORAGE GROUP TO root.nodatagroup2");
+      statement.execute("SET STORAGE GROUP TO root.nodatagroup3");
+      statement.execute("FLUSH root.nodatagroup1");
+      statement.execute("FLUSH root.nodatagroup2");
+      statement.execute("FLUSH root.nodatagroup3");
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
`e.printStackTrace()` can be replaced with logger

Reply:
Thanks for your friendly reminder.